src/main/java/com/aws/greengrass/clientdevices/auth/metrics/MetricsEmitter.java

```diff
@@ -29,16 +28,27 @@ public MetricsEmitter(ScheduledExecutorService ses, ClientDeviceAuthMetrics metr
         this.metrics = metrics;
     }
 
+    /**
+     * Restart the metrics emitter and apply configurations.
+     *
+     * @param periodicAggregateIntervalSec Periodic aggregate interval in seconds
+     */
+    public void restart(int periodicAggregateIntervalSec) {
```

Comment:
nit: this is the same as start, so we can remove this---just add a comment in start that mentions it'll kill previously running tasks.

Reply:
updated it to just use start()